src/docs/issue-platform-detectors.mdx

```diff
@@ -0,0 +1,128 @@
+
+# Bits’n’Bobs
+
+Performance issues affect many parts of the system, and it’s easy to overlook something. Here are a few places to look to make sure your issues are working well:
+
+- UI edges. Do your issues look correct in the issue stream? Check the severity level, title, description, and hover card content.
+- Email notifications. Does the UI match the issue details page for your new detector’s issues?
+- Slack notifications. Do the notifications for your issue make sense?
+- Integrations. Does converting your issue to a Jira/Linear/GitLab task work nicely?
+- Searching. Is is possible to search for your issue by type in the issue stream?
+- Alpha and Beta badges.  Did you add Alpha or Beta badges to the issue details page as appropriate?
+- Public documentation. Does your detector have a corresponding page on [docs.sentry.io](https://docs.sentry.io)?
+
+# Testing
+
+Whenever possible, conduct tests using real data. It's advised to use actual transaction event JSON files to perform unit testing for Performance Detectors.
+
+Another tool that we used for validating and testing detector changes is a CLI script: https://github.com/getsentry/sentry/pull/39727
+
+# Deployment
+
+A safe rollout strategy will depend on your team's implementation of detection and issue creation. However, there are general considerations to keep in mind:
+
+- Dry run detectors
+    - Validate detection accuracy on real data
+    - Check detection in the production environment
+- Decouple detection and issue creation
+    - Roll out detection and issue creation separately
+- Options for rates, thresholds
+    There are two kinds of options we use at Sentry, options usually refers to “system-wide” `options`, and `projectoptions` refers to options set for specific projects. We are using a combination of these to allow us to opt in the following:
+
+    - Detection rates
+        - We have implemented separate detection rates, allowing us to add new detectors or adjust thresholds on existing detectors. This enables us to track and report these changes through metrics
+    - Thresholds
+        - This capability allows us to fine-tune our detectors and observe their effects in real-time within the production environment. By adjusting the detection parameters and thresholds, we can monitor how these changes impact the detection accuracy and the number of detected issues
+
+    ```python
+    # projectoptions/defaults.py
+    key="sentry:performance_issue_creation_rate", default=1.0
+
+    key="sentry:performance_issue_settings",
+    default={
+    "n_plus_one_db_count": 5,
+    "n_plus_one_db_duration_threshold": 500,
+    }
+    ```
+    ![Untitled](img/issuePlatformFingerprinting02.png)
+
+- Per-project detection settings
+
+    [Project settings:](https://sentry.io/settings/sentry/projects/sentry/performance/)
+
+    ![Untitled](img/issuePlatformFingerprinting03.png)
```

Comment:
complete nit: Alt titles make it easier when editing w/o looking at the rendered